app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -31,7 +31,7 @@ def perform(form526_submission_id)
 
           return if bp_readings(client).blank?
 
-          pdf = pdf(form526_submission.full_name, bp_readings(client), medications(client))
+          pdf = pdf(patient_info(form526_submission), bp_readings(client), medications(client))
```

Comment:
Update the parameter name of `pdf` method to be `patient_info` instead of `full_name`.